StreamChat-XCFrameworks.podspec

```diff
@@ -1,5 +1,5 @@
 Pod::Spec.new do |spec|
-  spec.name = "StreamChat-OSS"
+  spec.name = "StreamChat-XCFrameworks"
```

Comment:
Weird git diff. I guess it mixed it up by the fact that StreamChat.podspec was pointing to XCFrameworks before...